server/models/handlers/worker.js

```diff
@@ -0,0 +1,11 @@
+const workerModel = require('../db/worker');
+
+// Adds a timeslot into the database and returns true upon successful completion. 
```

Comment:
Hi Simran,
I would recommend revising this comment to avoid confusion as this method returns the worker objects and does not add a timeslot into the database.

Reply:
Hi Amy! 

I've rewritten the comment! Thanks a lot! 

Simran